### PR TITLE
Unit test for "ImgTitleDescription" component

### LINKED ISSUE #124: (SP: 1) Write unit test for "ImgTitleDescription" component
### Component unit test
Unit test for "**ImgTitleDescription**" component
Scenaries descriptions:
- [ ] Check for render image
- [ ] Check for render title
- [ ] Check for render description

[Link to component](https://github.com/Space2Study-Project/Space2Study-FrontEnd/blob/develop/src/components/img-title-description/ImgTitleDescription.jsx)

Current coverage:
Image

### PR DESCRIPTION
**Pull request by ArtemHolikov**
close #124 

- [x] Test should check image, title and description render
image
image

### CODE REVIEW

**Review by LeonidShamarin — APPROVED** [review]

**Review by marianamartsy — APPROVED** [review]
👍

**Review by AndriiNasadiuk — APPROVED** [review]

**Review by Mygal-Alex — APPROVED** [review]
👍👍👍👍

File: tests/unit/components/img-title-description/ImgTitleDescription.spec.jsx
Comment (vlad-khrychov) [inline tests/unit/components/img-title-description/ImgTitleDescription.spec.jsx:1]: Try to mock `TitleWithDescription`. We already have test on this component. 
Mock can be like `({title, description})=><div><span>title</span><span>description</span></div>`

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]
Everything is good, but we must mock all internal components

File: tests/unit/components/img-title-description/ImgTitleDescription.spec.jsx
Comment (ArtemHolikov) [reply tests/unit/components/img-title-description/ImgTitleDescription.spec.jsx:1]: Mocked. Is everything good now?
image

**Pull request merged**